src/lib/ketrew_engine.ml

```diff
@@ -725,7 +718,7 @@ module Run_automaton = struct
       return (make_new_target ~log:("Attempt to start") starting_attemp)
     | `Eval_condition (condition, make_new_target) ->
       begin
-        Target.Condition.eval condition
+        Ketrew_eval_condition.bool condition
```

Comment:
What about renaming `Ketrew_eval_condition` to `Ketrew_condition` and then the function `bool` to `eval` ? That seems to me more in line with what it's doing, in a literate way.